cypress.json

```diff
@@ -1 +1,3 @@
-{}
+{
+  "baseUrl": "http://localhost:3333"
```

Comment:
is this needed? if so, should  it be `localhost:8443` ?

Reply:
No, this `3333` is the webserver for this project